pygerm/reduction_old.py

```diff
@@ -0,0 +1,143 @@
+import pandas as pd
```

Comment:
if we don't need this just delete it.  If we do need this, keep it where it was please.

Reply:
ok i'll delete most, ive written comments as a review as to why im removing/keeping stuff there.